pom.xml

```diff
@@ -1479,16 +1479,6 @@
 
   <repositories>
     <repository>
-      <id>jboss-snapshots</id>
-      <url>https://repository.jboss.org/nexus/content/repositories/snapshots</url>
-      <snapshots>
-        <enabled>true</enabled>
-      </snapshots>
-      <releases>
-        <enabled>false</enabled>
-      </releases>
-    </repository>
-    <repository>
```

Comment:
> Having it there it allows someone to build one module only, but that is probably not required.
As I believe the coupling between modules is pretty tight, I wouldn't recommend building one module without building the dependent modules first.